types/office-js-preview/index.d.ts

```diff
@@ -953,27 +653,12 @@ declare namespace Office {
              *
              * @remarks
              * 
-             * <table>
-             *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}</td><td>Restricted</td></tr>
-             *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or Read</td></tr>
-             * </table>
-             * 
-             * **Support details**
+             * **{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}:** Restricted
```

Comment:
Hmm, this prop is weird. By keeping just the Outlook info, it seems like it's only supported by Outlook and not the other hosts.

Reply:
Follow up: is the Source property only an Outlook thing? Or was that table inaccurate?

Reply:
I can't say if source is Outlook only. Looks that way.